cdk/config.ts

```diff
@@ -20,6 +21,8 @@ export class Config {
     this.dataAccessRoleArn = process.env.DATA_ACCESS_ROLE_ARN!;
     if (!process.env.STAC_API_INTEGRATION_API_ARN) throw Error("Must provide STAC_API_INTEGRATION_API_ARN");
     this.stacApiIntegrationApiArn = process.env.STAC_API_INTEGRATION_API_ARN!;
+    if (!process.env.DB_ALLOCATED_STORAGE) throw Error("Must provide DB_ALLOCATED_STORAGE");
```

Comment:
The change to have it as an environment variable is good but I think keeping a default is fine; I think it was the same comment as the last PR

Reply:
It doesn't hurt to require with without any defaults -- it's because of a default that I myself made the mistake of provisioning too much storage. 

Although ofc it's much less likely to be overlooked if it's in a config option.